agents/check_mk_agent.aix

```diff
@@ -124,7 +124,9 @@ function run_cached {
         if (( $MTIME < $MAXAGE )) ; then
```

Comment:
Must the calulation of MTIME be adapted, too?

Reply:
Hi, this is not needed.